include/OpenSpaceToolkit/Core/Types/Real.hpp

```diff
@@ -116,6 +116,8 @@ class Real
 
         types::String           toString                                    (   const   types::Integer&             aPrecision                                  =   types::Integer::Undefined() ) const ;
 
+        Real::ValueType         toFloat                                     ( ) const ;
```

Comment:
May not be necessary, since converter had been specified.

To transform `Real` to `double`, do:

```cpp
static_cast<double>(myReal)
```

Reply:
Indeed, the `static_cast` functions are sufficient. Implemented for `Integer` and `Real` and removed the C++ temporary getter functions